setup.cfg

```diff
@@ -1,3 +1,5 @@
 [tool:pytest]
 xfail_strict = true
 faulthandler_timeout=60
+markers =
+    upstream_only: tests that only make sense for upstream to run
```

Comment:
Can you explain this a little more here? Maybe something like "tests that may fail spuriously if Trio's dependencies are updated"

Reply:
Or even "Tests that should be skipped by downstream redistributors."

And potentially the marker's name would be even clearer if it was `redistributors_should_skip`?

Reply:
I've adjusted this to exactly the wording @njsmith suggested.